src/Raven.Server/Rachis/CandidateAmbassador.cs

```diff
@@ -37,8 +37,8 @@ public string StatusMessage
         public volatile int ClusterCommandsVersion; // default is 0
         public string Tag => _tag;
 
-        private RemoteConnection _connection;
-        private RemoteConnection _publishedConnection;
+        private RemoteConnection.RemoteConnection _connection;
```

Comment:
why all these changes?

Reply:
I moved some of the code (the riders) in RemoteConnection file to another file and put them both in a directory called RemoteConnection